### PR TITLE
docs/faq-stock-tools-icons

### PR DESCRIPTION
**Pull request by DJTechnoo**
Added new point for external file references in the FAQ.

### CODE REVIEW

File: docs/getting-started/frequently-asked-questions.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ For basic usage, Highcharts doesn't refer to any files other than highcharts.js/
...
 *   Certain features, including client side exporting, may require third-party dependencies. Some of these are loaded on demand from our servers, but in these cases the loading location is configurable. Details on all external dependencies, including licensing and security details, can be found in the [optional dependencies](https://highcharts.com/docs/getting-started/optional-dependencies) documentation article.
+* Stock Tools icons. By default the icons are loaded dynamically from https://code.highcharts.com. To change where icons should be loaded from see the [iconsURL](https://api.highcharts.com/highstock/stockTools.gui.iconsURL) option.
```
Comment (pawelfus) [inline docs/getting-started/frequently-asked-questions.md:25]: wrong URL, this one leads to deprecated option, please use: https://api.highcharts.com/highstock/navigation.iconsURL

**Review by pawelfus — CHANGES_REQUESTED** [review]

**Review by kamil-musialowski — APPROVED** [review]
Apart from Pawel's comment - all good!

**Review by bm64 — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by pawellysy — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**